dist/src/main/java/io/camunda/zeebe/gateway/GatewayModuleConfiguration.java

```diff
@@ -35,20 +37,23 @@
 @Configuration(proxyBeanMethods = false)
 @ComponentScan(
     basePackages = {
-      "io.camunda.commons",
```

Comment:
❓ this was removed as we have now the CommonsConfiguration component right?

Reply:
Yep, I wanted to avoid adding that base package everywhere. That's why I moved to the `CommonsModuleConfiguration`, which will do the scanning there.